ComfortZoneApp/app/src/main/java/com/example/comfortzone/AllWeathersDatabase.java

```diff
@@ -20,7 +20,7 @@ public abstract class AllWeathersDatabase extends RoomDatabase {
     public static AllWeathersDatabase getDbInstance(Context context) {
         if (context == null || INSTANCE == null) {
             INSTANCE = Room.databaseBuilder(context.getApplicationContext(), AllWeathersDatabase.class, DB_NAME)
-                    .allowMainThreadQueries().build();
+                    .allowMainThreadQueries().fallbackToDestructiveMigration().build();
```

Comment:
when i change database versions, room uses migrations to upgrade the version, but if theyre not found, the app will crash, so adding fallbackToDestructiveMigration recreates the table so that it wont crash